bluebottle/bb_projects/models.py

```diff
@@ -193,24 +193,11 @@ class BaseProject(models.Model, GetTweetMixin):
         related_name="project_account_holder_country")
 
     # Bank details
-    account_iban = IBANField(_("account IBAN"), null=True, blank=True)
+    account_number = models.CharField(_("Account number"), max_length=255, null=True, blank=True)
```

Comment:
Yeah, nut it does not match exactly. There it is called account_iban. Which is probably wrong.